### PR TITLE
SPEC file updated to version 0.10.0 pre-release

### PR DESCRIPTION
**Pull request by xambroz**
First round of the updates for the radare SPEC file ... to make it somehow build RPM package with the current version of HEAD.

### CODE REVIEW

File: radare2.spec [hunk 1]
```diff
@@ -1,15 +1,30 @@
+%global         gituser         radare
+%global         gitname         radare2
+#%global        commit          a093958b6d24015d82782eb20a2e10d8f4afcd85
```
Comment (krytarowski) [inline radare2.spec]: Bug, RPM `#%` still evaluates values commented out. Go for `#%%`, the same applies later.
Why `%global`?

**Comment by krytarowski** [comment]
What's the point of RPM, if it's not portable across RPM systems, OpenMandriva and Mageia aren't compatible. Fedora and Red Hat are closer. Different story with OpenSUSE. I'm not counting minor distros.

**Comment by xambroz** [comment]
re krytarowski
> What's the point of RPM, if it's not portable across RPM systems, 
> OpenMandriva and Mageia aren't  compatible.
> Different story with OpenSUSE. I'm not counting minor distros.
Please note that the RPM spec file for Fedora/RHEL was already in the package. I am just fixing it to the current version.
RPM format allows for ifdef-else constructions based on distribution so it could be possible to make single SPEC file for Fedora & Mandriva ... I am not expert there so I will leave it on Mandriva specialist.
Same applies for OpenSUSE. 

> Fedora and Red Hat are closer. 
I am usually using single spec file for both Fedora and EPEL7 (=RHEL7) in all packages I am maintaining ... no problem there.

Best regards
Michal Ambroz

**Comment by radare** [comment]
Thanks! I have merged the squashed commit. lgtm

**Pull request closed without merging**